pkl-formatter/src/test/files/FormatterSnippetTests/output/dangling-doccomment.pkl

```diff
@@ -0,0 +1,8 @@
+module dangling
+
+/// Doc comment.
+///
+/// for this field
+// sepearted by a stray comment
```

Comment:
Unsure for the fix here as well: most often the correct thing to do would be replacing `//` with `///`, but that brings us in the territory of fixing, not formatting.